core/server/proxy/src/main/java/alluxio/proxy/s3/CompleteMultipartUploadHandler.java

```diff
@@ -161,15 +170,19 @@ public void handle(String s, Request request, HttpServletRequest httpServletRequ
             httpServletResponse.setStatus(s3Exception.getErrorCode().getStatus().getStatusCode());
           }
         }
-        LOG.error(e.toString());
+        LOG.error(ThreadUtils.formatStackTrace(cause));
       }
       httpServletResponse.getWriter().flush();
       request.setHandled(true);
     } catch (Exception e) {
       // This try-catch is not intended to handle any exceptions, it is purely
       // to ensure that encountered exceptions get logged.
-      LOG.error("Unhandled exception for {}. {}", s, e);
+      LOG.error("Unhandled exception for {}. {}", s, ThreadUtils.formatStackTrace(e));
```

Comment:
I think if you want to print the stack trace you you just do something like this
LOG.error("Unhandled exception for {}.", s, e);

Reply:
yeah i would like to do some other customized things like cap stack trace layer in the util func in the future, for now would like to print full thing for some debugging purpose.